src/api/main.py

```diff
@@ -170,9 +170,8 @@ def default(self, obj):
 def api_map():
     """List endpoints for api."""
     endpoints = {
-        endpoint.rule: endpoint.methods
-        for endpoint in app.url_map.__dict__["_rules"]
-        if endpoint.rule not in ["/static/<path:filename>", "/"]
+        k: f"{v[0].methods}  {v[0].rule}"
```

Comment:
Is the first element the only populated/important one? If so please add a comment so the reasoning is clear.

Reply:
@mcdonnnj the way its set up, each value `v` is a list with only one element. so i pull the first index so i can access the properties of that element. i'll leave a comment shortly

Reply:
resolved with [ca4227f](https://github.com/cisagov/domain-manager-api/pull/515/commits/ca4227f70006a7001f215050295e5a2d99fa1971)